nautobot_circuit_maintenance/handle_notifications/sources.py

```diff
@@ -534,7 +534,7 @@ class GmailAPI(EmailSource):
     credentials_file: str
     account: str
```

Comment:
For my education, why doesn't `account` need a `repr=False` declaration?

Reply:
I considered the account as not so sensitive, I could include, for sure

Reply:
Sorry, reading failure on my part :)